README.md

```diff
@@ -368,6 +368,12 @@ az spring-cloud app append-loaded-public-certificate \
     --certificate-name ${CLIENT_SSL_CERTIFICATE_NAME} --load-trust-store true
 ```
 
+> [!Reminder]
+> Currently the auto-wiring of “--load-trust-store” is not compatible with java apps which requires build service, that includes java apps deployed from source code and all java apps in enterprise tier. We are now working on supporting these apps and it would be coming soon.
```

Comment:
Suggest simply rephrasing to "Currently "--load-trust-store" parameter is not supported in Enterprise tier."